pages/common/pamfix.md

```diff
@@ -0,0 +1,17 @@
+# pamfix
+
+> Fix errors in Netpbm files.
+> See also: `pamfile`, `pamvalidate`.
+> More information: <https://netpbm.sourceforge.net/doc/pamfix.html>.
+
+- Fix a Netpbm file that is missing its last part:
+
+`pamfix -truncate {{path/to/corrupted.pam}} > {{path/to/output.pam}}`
+
+- Fix a Netpbm file where pixel values exceed the image's maxval by lowering the offending pixels' values:
+
+`pamfix -clip {{path/to/corrupted.pam}} > {{path/to/output.pam}}`
+
+- Fix a Netpbm file where pixel values exceed the image's maxval increasing the maxval:
+
+`pamfix -changemaxval {{path/to/corrupted.pam}} > {{path/to/output.pam}}`
```

Comment:
```suggestion
`pamfix -changemaxval {{path/to/corrupted.pam|pbm|pgm|ppm}} > {{path/to/output.pam|pbm|pgm|ppm}}`
```

Reply:
```suggestion
`pamfix -changemaxval {{path/to/corrupted.pam|pbm|pgm|ppm}} > {{path/to/output.pam|pbm|pgm|ppm}}`
```

I don'tthink tldr has this syntax. I'd rather to this.

Reply:
> I don'tthink tldr has this syntax. I'd rather to this.

We use this in quite a few places as a replacement to `.ext` in placeholders. I was planning to document this in the style guide with https://github.com/tldr-pages/tldr/issues/12396.

Reply:
You are right, we can introduce this but we should document it.